package.json

```diff
@@ -21,7 +21,6 @@
     },
     "homepage": "https://github.com/chicagomaroon/chicagomaroon.github.io#readme",
     "devDependencies": {
-        "@amcharts/amcharts5": "^5.3.4",
```

Comment:
Since AmCharts is being imported from an external source, there's no need to have it as a dependency.